pom.xml

```diff
@@ -91,23 +91,6 @@
                 </exclusion>
             </exclusions>
         </dependency>
-        
```

Comment:
Moved these to the bottom of the dependencies list to make it easier to compare our fork to the upstream repo.